### PR TITLE
Create - 2023-11-01-KANDYKORN.md

### PR DESCRIPTION
**Pull request by aholman0725**

### CODE REVIEW

File: content/attacks/2023-11-01- KANDYKORN [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- APT-43
+title: "North Korean Hackers Targeting Crypto Experts with KANDYKORN macOS Malware."
+loss: No monetary loss
+---
+
+## Summary
+
+State-sponsored threat actors from the Democratic People's Republic of Korea (DPRK) have been found targeting blockchain engineers of an unnamed crypto exchange platform via Discord with a novel macOS malware dubbed KANDYKORN.
+Elastic Security Labs said the activity, traced back to April 2023, exhibits overlaps with the infamous adversarial collective Lazarus Group, citing an analysis of the network infrastructure and techniques used.
+"Threat actors lured blockchain engineers with a Python application to gain initial access to the environment," security researchers Ricardo Ungureanu, Seth Goodwin, and Andrew Pease said in a report published today.
```
Comment (jhirschkorn) [inline content/attacks/2023-11-01- KANDYKORN:24]: add/embed links for quotes/references whenever possible

File: content/attacks/2023-11-01- KANDYKORN [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- Execution flow hijacking
+- defensive evasion techniques
+- KANDYKORN
+- Swift-based self-signed binary
+- APT-43
+title: "North Korean Hackers Targeting Crypto Experts with KANDYKORN macOS Malware."
+loss: No monetary loss
+---
+
+## Summary
```
Comment (jhirschkorn) [inline content/attacks/2023-11-01- KANDYKORN:20]: a little bit too much discourse/discussion. focus the summary on the hack/exploit, losses, entities involved.  No need to include extended discussion on peripherals.

File: content/attacks/2023-11-01- KANDYKORN [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- No monetary loss
+
+## Timeline
+
+The hacks can be traced back to April 2023.
+
+## Security Failure Causes
+
+Threat actors lured blockchain engineers with a Python application to gain initial access to the environment.
+This intrusion involved multiple complex stages that each employed deliberate defense evasion techniques.
```
Comment (jhirschkorn) [inline content/attacks/2023-11-01- KANDYKORN:57]: delete this line

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

File: content/attacks/2023-11-01- KANDYKORN (hunk 3 above)
Comment (aholman0725) [reply content/attacks/2023-11-01- KANDYKORN:57]: Delete line 56 or 57?

File: content/attacks/2023-11-01- KANDYKORN [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+- North Korean Lazarus Group is suspected to be behind the theft.
+- State-sponsored threat actors from the Democratic People's Republic of Korea.
+
+## Losses
+
+- No monetary loss
+
+## Timeline
+
+The hacks can be traced back to April 2023.
```
[the remark this answers is not in the document]
Comment (aholman0725) [reply content/attacks/2023-11-01- KANDYKORN:52]: Should I include links in the timeline?

[the remark this answers is not in the document]
Comment (aholman0725) [reply content/attacks/2023-11-01- KANDYKORN]: The updated timeline contains as much info as I could find without going into cybersecurity and software vulnerability.

[the remark this answers is not in the document]
Comment (jhirschkorn) [reply content/attacks/2023-11-01- KANDYKORN]: include links in the timeline/article as appropriate

File: content/attacks/2023-11-01- KANDYKORN (hunk 3 above)
Comment (jhirschkorn) [reply content/attacks/2023-11-01- KANDYKORN]: 57

Comment (aholman0725) [reply content/attacks/2023-11-01- KANDYKORN]: Done. I changed this pull request up a bit. I look forward to your feedback.

File: content/attacks/2023-11-01- KANDYKORN [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+
+-There is No monetary loss reported.
+
+
+## Timeline
+
+
+**April 2023:" Elastic Security Labs [traced](https://www.elastic.co/security-labs/elastic-catches-dprk-passing-out-kandykorn) the Lazarus APT group's campaign using the KandyKorn malware through the RC4 key used for encryption in the SUGARLOADER and KandyKorn C2. The threat is ongoing, and there is ongoing development of tools and techniques by the attackers.
+**Late July 2023:** There a subReditts user reports being contacted to solve a Python coding [challenge](https://www.elastic.co/security-labs/elastic-catches-dprk-passing-out-kandykorn.) appearing to be for a speedtest.
```
Comment (jhirschkorn) [inline content/attacks/2023-11-01- KANDYKORN:38]: Remove "Late"

File: content/attacks/2023-11-01- KANDYKORN [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Losses
+
+- No monetary loss
+
+## Timeline
+
+The hacks can be traced back to April 2023.
+
+## Security Failure Causes
```
[the remark this answers is not in the document]
Comment (jhirschkorn) [reply content/attacks/2023-11-01- KANDYKORN:42]: bump

File: content/attacks/2023-11-01- KANDYKORN (hunk 5 above)
Comment (aholman0725) [reply content/attacks/2023-11-01- KANDYKORN]: Removed.

**Comment by aholman0725** [comment]
@evgenydmitriev  I believe I have made all requested changes. Please let me know if I missed something.

**Comment by evgenydmitriev** [comment]
@aholman0725, here are some reason this and other pull requests from you are not being reviewed:

- You are adding multiple commits and comments in succession, so it's hard to tell when something is a work in progress and when you want us to review the work. Advice: read up on the [pull request process](https://docs.github.com/en/pull-requests) and use [draft feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request).
- You are leaving reviewers' comments unaddressed. [Example](https://github.com/1712n/dn-institute/pull/305#discussion_r1392863693). Make sure you mark conversations as resolved if you addressed the comment. Also, make sure you address mentioned issues in other open pull requests you have opened.
- Grammar, punctuation, style, best practices. I already [mentioned](https://github.com/1712n/dn-institute/pull/319#pullrequestreview-1757453224) this multiple times, you either need to pay attention to those yourself, or ask GPT-4 or Bard to help you.
- In this specific pull request, I see problems with style, markdown usage, as well as following our guidelines on target entities and losses. This wiki is focused on critical blockchain ecosystem attacks, not exploits of individual developers with no losses. 

Closing this PR and giving it another week for other pull requests to be brought up to the level where they can be merged.

**Comment by aholman0725** [comment]
@evgenydmitriev Thank you for the feedback. I understand the commit issue. Would it be easier if I started the current pull requests from scratch? I'll make sure to use the draft feature moving forward. ChatGPT4 is currently unavailable. Is there another bot you suggest using? I'll address the other issues as well. I appreciate your patience and assistance.

**Pull request closed without merging**